packages/atti-components/src/components/Slider/theme/index.js

```diff
@@ -0,0 +1,55 @@
+export default ({borders, colors, spaces}) => ({
+  sliderRail: {
+    borders: {
+      radius: borders.radius.small,
+    },
+    colors: {
+      backgroundColor: colors.darkPrimary,
+    },
+    sizes: {
+      height: spaces.s0_25,
+      width: spaces.s16,
+    },
+  },
+  sliderIndicator: {
+    borders: {
+      radius: borders.radius.full,
+    },
+    colors: {
+      backgroundColor: colors.primary,
+    },
+    sizes: {
+      height: spaces.s0_5,
+      width: spaces.s0_5,
+    },
+    shadows: {
+      hover: '0px 0px 0px 4px rgba(18,43,181, 0.4)',
+      moving: '0px 0px 0px 8px rgba(18,43,181,0.16)',
```

Comment:
I wonder if we should better use the `shadows` of the `theme` and/or create ones to unify the shadows style across the library 🤔

Reply:
It is really specific for this component. It's why I choose to not add it to the theme, if you think it might be better I still can add it as a part of the theme.

Reply:
🤔 Most of UI libraries like MUI use the same shadow values in any of their components to create a consistent styleguide - and it makes sense so I think we should do the same. Except if as you say the shadows of your component are too specific.... It's up to you :)

Reply:
I'll keep it inside the component, it is really unlikely that it's going to be re-used anywhere else.